### DISCUSSION TITLE
Cancel a subscription without an SaaS entry

### DISCUSSION THREAD

**Discussion by Chanticrow** [rcl-ssl/docs#16] [General]
While setting up a new environment we were exploring using RCL for certs, but ended up using the app service certs instead.  I guess we got far enough in the RCL setup that we created a subscription, but never finished creating the resource.  I do not have anything from RCL listed in my SaaS area.  

How do I cancel this subscription?

**Comment by rcladmin** [comment]
You can follow the instruction in this link to cancel a subscription:

https://docs.common.rclapp.com/subscription/subscription.html#unsubscribe-for-a-rcl-app

**Reply by Chanticrow** [reply]
I tried that, but I do not have the RCL SSL app listed in my SaaS area in Azure.  

I could try creating a new one, and then cancelling it.  Would that set up a second subscription, or would it recognize my existing subscription and use it?

**Reply by rcladmin** [reply]
Once you did not activate the subscription on the RCL website, you will not be charged. Please do not create a new subscription. Can you send me a screenshot of your SaaS page so that I can understand what is going on.

**Reply by Chanticrow** [reply]
I am getting charged, according to my Azure invoice.  :D  That's why I'm posting.  I'll provide some additional info.

**Reply by Chanticrow** [reply]
Here's a shot of my SaaS panel, and a snip from my latest invoice.

https://imgur.com/a/mm9qsEt

**Reply by rcladmin** [reply]
What is the subscription on the the invoice and what is the subscription on the SaaS page. Maybe, the SaaS page is a different subscription to the invoice.

Can you tell me what is the username/email you subscribe to the RCL app, I will check the subscription on the RCL App for you.

**Reply by Chanticrow** [reply]
Would you contact me via email for this?  I would rather not post Azure related account info on an open forum.

**Reply by jeremydumais** [reply]
I had the same problem on my side and I found the entry in SaaS (classic) instead of SaaS directly.

**Reply by Chanticrow** [reply]
To follow up on this for future readers, I ended up on a screen sharing call with Microsoft support with one of their portal techs.  We stepped through every screen and billing area and, in the end, even Microsoft could not find the subscription.  They could see it on their side, but there was no way to access it from my perspective.  They were able to cancel it from their side.
